populate_db.py

```diff
@@ -222,9 +222,13 @@ def create_admin_message_settings():
                      "New Session Proposal"]
     for mail in MAILS:
         if mail in default_mails:
-            get_or_create(MessageSettings, action=mail, mail_status=1, notification_status=1, user_control_status=1)
+            get_or_create(MessageSettings, action=mail, mail_status=True,
```

Comment:
`mailstatus=1` vs `True` - what does it mean in this context? A comment would be helpful :)